src/views/networkpolicies/list/EnableMultiPage.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { FC } from 'react';
+import { useNavigate } from 'react-router-dom-v5-compat';
+
+import { modelToGroupVersionKind, modelToRef } from '@kubevirt-ui/kubevirt-api/console';
+import {
+  k8sPatch,
+  K8sResourceCommon,
+  K8sVerb,
+  useAccessReview,
+  useK8sWatchResource,
+} from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Button,
+  EmptyState,
+  EmptyStateActions,
+  EmptyStateFooter,
+  EmptyStateHeader,
+  Tooltip,
+} from '@patternfly/react-core';
+import { useLastNamespacePath } from '@utils/hooks/useLastNamespacePath';
+import { useNetworkingTranslation } from '@utils/hooks/useNetworkingTranslation';
+import { MultiNetworkPolicyModel } from '@utils/models';
+import { NetworkConfigModel } from '@views/nads/form/utils/constants';
+
+import '@styles/list-management-group.scss';
```

Comment:
what is this for?

Reply:
it's on all list pages. It add some style that is on utils/style